pontoon/translations/views.py

```diff
@@ -126,7 +126,11 @@ def create_translation(request):
         )
 
     # When user makes their first contribution to the team, notify team managers
-    first_contribution = not project.system_project and user.is_new_contributor(locale)
+    first_contribution = (
+        not project.system_project
+        and user.is_new_contributor(locale)
+        and user != project.contact
```

Comment:
Probably imperceptible optimization, but checking the project data doesn't need a DB query.
```suggestion
        and user != project.contact
        and user.is_new_contributor(locale)
```